### PR TITLE
Add previous page / next page buttons to docs

### PR DESCRIPTION
**Pull request by lfades**

### CODE REVIEW

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**/docs/\[...slug\]**](https://next-site-git-docs-navigation-btns.zeit.now.sh/docs/[...slug]) |
|:-:|
|<a href="https://next-site-git-docs-navigation-btns.zeit.now.sh/docs/[...slug]">Screenshot of /docs/[...slug]</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/UWKR4GhSAkcc_zYY9KpdOrTSjLo_">(view full size)</a> |







Commit 30a270b7be5cae870d1450dea44b417ab16acfa7 (https://next-site-git-docs-navigation-btns.zeit.now.sh).

**Comment by timneutkens** [comment]
@chibicode could you review this change 🕵

File: components/arrow-icon.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const ArrowIcon = ({ left, right, children }) => (
```
Comment (chibicode) [inline components/arrow-icon.js]:
```suggestion
const ArrowIcon = ({ left, right, children, flex }) => (
```

File: components/arrow-icon.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <span>
+    {children}
+    <style jsx>{`
+      span {
+        ${left ? 'margin-right: .5rem; margin-left: -.25rem;' : ''}
+        ${right ? 'margin-left: .5rem; margin-right: -.25rem;' : ''}
+        display: inline-block;
```
Comment (chibicode) [inline components/arrow-icon.js]:
```suggestion
        display: ${flex ? 'inline-flex' : 'inline-block'};
```

File: components/docs/docs-page.js [hunk 3]
```diff
@@ -1,30 +1,64 @@
...
         available <a href="/docs/old">here</a>.
       </Notification>
+
       {/* eslint-disable-next-line */}
       <div dangerouslySetInnerHTML={{ __html: html }} />
+
+      <div className="page-nav">
+        {prevRoute ? (
+          <Button href={href} as={removeFromLast(prevRoute.path, '.')}>
+            <ArrowIcon left>
```
Comment (chibicode) [inline components/docs/docs-page.js]:
```suggestion
            <ArrowIcon left flex>
```

File: components/docs/docs-page.js [hunk 4]
```diff
@@ -1,30 +1,64 @@
...
+            </ArrowIcon>
+            {prevRoute.title}
+          </Button>
+        ) : (
+          <span />
+        )}
+        {nextRoute && (
+          <Button href={href} as={removeFromLast(nextRoute.path, '.')}>
+            {nextRoute.title}
+            <ArrowIcon right>
```
Comment (chibicode) [inline components/docs/docs-page.js]:
```suggestion
            <ArrowIcon right flex>
```

File: components/arrow-icon.js (hunk 2 above)
Comment (chibicode) [reply components/arrow-icon.js]: @lfades I think line 8 (`display: inline-block`) needs to be removed https://github.com/zeit/next-site/pull/543/files#diff-62d073512f117fe3c715af3e58ee56caR8

File: components/arrow-icon.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+const ArrowIcon = ({ left, right, children, flex }) => (
+  <span>
+    {children}
+    <style jsx>{`
+      span {
+        ${left ? 'margin-right: .5rem; margin-left: -.25rem;' : ''}
+        ${right ? 'margin-left: .5rem; margin-right: -.25rem;' : ''}
+        display: inline-block;
```
Comment (chibicode) [inline components/arrow-icon.js]: @lfades I think line 8 (`display: inline-block`) needs to be removed https://github.com/zeit/next-site/pull/543/files#diff-62d073512f117fe3c715af3e58ee56caR8

**Comment by chibicode** [comment]
@lfades @timneutkens this looks good to me, can we merge it?

**Pull request merged**